icu4c/source/data/misc/supplementalData.txt

```diff
@@ -35597,398 +35597,297 @@ supplementalData:table(nofallback){
+                }
+            }
+            _default{
+                001{
+                    square-kilometer{
+                        geq:1{""}
+                    }
+                    hectare{
+                        geq:1{""}
+                    }
+                    square-meter{
+                        geq:1{""}
+                    }
+                    square-centimeter{""}
+                }
+            }
         }
-        US{
-            area-land-agricult{"acre"}
-            area-land-commercl{"acre"}
-            area-land-residntl{"acre"}
-            consumption-vehicle-fuel{"mile-per-gallon"}
-            length-person{"inch"}
-            length-person-informal{"foot inch"}
-            length-person-small{"inch"}
-            length-rainfall{"inch"}
-            length-road{"mile"}
-            length-road-small{"foot"}
-            length-snowfall{"inch"}
-            length-vehicle{"foot inch"}
-            length-visiblty{"mile"}
-            length-visiblty-small{"foot"}
-            mass-person{"pound"}
-            mass-person-small{"pound ounce"}
-            pressure-baromtrc{"inch-hg"}
-            speed-road-travel{"mile-per-hour"}
-            speed-wind{"mile-per-hour"}
-            temperature-person{"fahrenheit"}
-            temperature-weather{"fahrenheit"}
-            volume-vehicle-fuel{"gallon"}
+        "duration"{
+            "music-track"{
+                001{
+                    minute+second{""}
+                }
+            }
+            "person-age"{
+                001{
+                    year-person+month-person{""}
+                }
+            }
+            "tv-program"{
+                001{
+                    minute+second{""}
+                }
+            }
+            _default{
+                001{
+                    year{
+                        geq:1{""}
+                    }
+                    month{
+                        geq:1{""}
+                    }
+                    day{
+                        geq:1{""}
+                    }
+                    hour{
+                        geq:1{""}
+                    }
+                    minute{
+                        geq:1{""}
+                    }
+                    second{
+                        geq:1{""}
+                    }
+                    millisecond{
+                        geq:1{""}
+                    }
+                    microsecond{
+                        geq:1{""}
+                    }
+                    nanosecond{""}
+                }
+            }
         }
-        VC{
-            concentr-blood-glucose{"millimole-per-liter"}
+	// DEMO: alt{"informal"}:
+        "length"{
+            "road"{
+                001{
+                    kilometer{""}
+                }
+                GB{
+                    mile{""}
+                }
+                SE{
+                    mile-scandinavian{
+                        alt{"informal"}
```

Comment:
Question for @macchiati: does alt="informal" imply that the fallback chain should be used or not?  For example, if SE has foot for small road distance (not alt), and alt="informal" mile-scandanavian, if the alt is requested, do we use feet or not?